aptos-move/aptos-gas/src/aptos_framework.rs

```diff
@@ -116,5 +116,8 @@ crate::natives::define_gas_parameters_for_natives!(GasParameters, "aptos_framewo
     [.aggregator.destroy.base, "aggregator.destroy.base", 500 * MUL],
     [.aggregator_factory.new_aggregator.base, "aggregator_factory.new_aggregator.base", 500 * MUL],
 
-    [.object.exists_at.base_cost, { 7.. => "object.exists_at.base" }, 250 * MUL]
+    [.object.exists_at.base_cost, { 7.. => "object.exists_at.base" }, 250 * MUL],
+    // These are dummy value, they copied from storage gas in aptos-core/aptos-vm/src/aptos_vm_impl.rs
```

Comment:
@vgao1996 do you have instructions for running the benchmarking tool you used to get CPU instruction count?

Reply:
https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/aptos-vm-profiling#readme